device-server/src/test/kotlin/com/badoo/automation/deviceserver/NodeConfigTest.kt

```diff
@@ -49,16 +53,16 @@ class NodeConfigTest {
         val config = JsonMapper().fromJson<NodeConfig>("{}")
 
         val expected = NodeConfig(
-            type = NodeConfig.NodeType.Simulators,
```

Comment:
Could you please revert unnecessary style changes, or put them into a separate commit/PR?